queries/beancount/locals.scm

```diff
@@ -0,0 +1,3 @@
+(transaction
+    (date) @definition.date
+    (txn_strings) @definition.payee) @definition.transaction
```

Comment:
At the moment, we only have a fixed set of definitions. You would have to edit contributing.md to add new queries.

Reply:
removed the locals file since this language does not have normal language mechanics